### ISSUE TITLE
Find a good formulation for latent heat storages

### ISSUE THREAD

**Issue by p-snft** [oemof/oemof-thermal#5]
The combination of heat pumps and phase change material can be used as a rather inexpensive and ecological seasonal energy storage (German: Eisspeicher). This is one component to be developed in this repository.

(This is a todo note for myself. Assign me when I become part of the team.)

**Comment by ckaldemeyer** [comment]
> The combination of heat pumps and phase change material can be used as a rather inexpensive and ecological seasonal energy storage (German: Eisspeicher). This is one component to be developed in this repository.
> 
> (This is a todo note for myself. Assign me when I become part of the team.)

I also looked into ice storage and it is actually quite interesting but on the other hand does not seem to be very prominent in the modelling/energy system analysis world.

What I ask myself is how this technology compares to sensible seasonal heat storages such as pit storages in DK e.g. this one: https://www.sciencedirect.com/science/article/pii/S0360544218305619?via=ihub

One big advantage I see would be the energy density and thus the application also in areas with limited space. Cost-wise I assume it to be more expensive.

Let me know if you figure out more! I will do so as well as I find it really interesting ;-)

**Comment by p-snft** [comment]
In fact, a first order approximation for an ice storage is just a combination of heat pumps and a low-temperature thermal storage at T=0°C=const. and a rather large storage capacity. So, due to the constant temperature, it should even be easier to model than non-latent heat storage.

PS: Unluckily, the paper is behind a paywall.

**Comment by ckaldemeyer** [comment]
> In fact, a first order approximation for an ice storage is just a combination of heat pumps and a low-temperature thermal storage at T=0°C=const. and a rather large storage capacity. So, due to the constant temperature, it should even be easier to model than latent heat storage.

Yes, I think in a simplified version you could model it like this when withdrawing heat. On the other hand you would have to melt it again and the component thus would need some heat inflow. So generally the component including the heat pump circuit could have one electricity input, one heat input (which could be anything e.g. solar thermal) and one heat output. The heat output and the electricity input are then linked via the COP. And depending on the technical system one could either charge or discharge the system but not do this simultaneously. At leat it would work like this in a sketched version from my point of view..

Concerning the Paywall just check Scihub or send me an e-mail ;-)
